Examples/Swift/FillPatternExample.swift

```diff
@@ -0,0 +1,59 @@
+import Mapbox
+
+@objc(FillPatternExample_Swift)
+
+class FillPattern_Swift: UIViewController, MGLMapViewDelegate {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Set the map’s size, style, center coordinate, and zoom level.
+        let mapView = MGLMapView(frame: view.bounds)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.styleURL = MGLStyle.darkStyleURL(withVersion: 9)
```

Comment:
Per @friedbunny's comment in https://github.com/mapbox/ios-sdk-examples/pull/94#discussion_r124073373, I originally had `MGLStyle.darkStyleURL()`, but it was suggested that we should pin the version to a specific number and explain that relying on simply the default version may become problematic if the current version differs significantly. Although, the `place-city-sm` label is an important layer that I feel wouldn't be drastically changed anytime soon, so I'm ok reverting it back to `MGLStyle.darkStyleURL()` , and adding more specific commentary instead.